.gitignore

```diff
@@ -16,7 +16,6 @@ data/
 agent/
 service/
 
-asylum_abci_app/
```

Comment:
yeah I was force adding this packages dir. It was here because the creation of the agent would result in a similarly named folder in the repo root, which we obviously don't want to add to git. Should use used `/asylum_abci_app/` instead of `asylum_abci_app/`